x-pack/plugins/spaces/public/space_selector/space_selector.tsx

```diff
@@ -135,21 +145,38 @@ export class SpaceSelector extends Component<Props, State> {
               <SpaceCards spaces={filteredSpaces} serverBasePath={this.props.serverBasePath} />
             )}
 
-            {!this.state.loading && filteredSpaces.length === 0 && (
+            {!this.state.loading && !this.state.error && filteredSpaces.length === 0 && (
               <Fragment>
                 <EuiSpacer />
-                <EuiText
-                  color="subdued"
-                  // @ts-ignore
-                  textAlign="center"
-                >
+                <EuiText color="subdued" textAlign="center">
                   <FormattedMessage
                     id="xpack.spaces.spaceSelector.noSpacesMatchSearchCriteriaDescription"
                     defaultMessage="No spaces match search criteria"
                   />
                 </EuiText>
               </Fragment>
             )}
+
+            {!this.state.loading && this.state.error && (
+              <Fragment>
+                <EuiSpacer />
+                <EuiPanel>
```

Comment:
To address the full width panel, we can adopt the same styles used on the login selector page:

```suggestion
                <EuiPanel className="spcSpaceSelector__errorPanel>
```

And then inside [`space_selector.scss`](https://github.com/elastic/kibana/blob/0fbbc788525fa0abc6b21ff40082c41b6528f0c6/x-pack/plugins/spaces/public/space_selector/space_selector.scss):

```scss
.spcSpaceSelector__errorPanel {
  position: relative;
  margin: auto;
  padding-left: $euiSizeXL;
  padding-right: $euiSizeXL;
  max-width: 700px;
}
```


This will give us:

![image](https://user-images.githubusercontent.com/3493255/95221989-08f31700-07c6-11eb-82e5-1eaa9a832fa8.png)